invenio/modules/editor/bundles.py

```diff
@@ -37,7 +34,17 @@
     "js/editor/marcxml.js",
     "js/editor/clipboard.js",
     output="editor.js",
+    filters="uglifyjs",
```

Comment:
what about using https://github.com/inveniosoftware/invenio/pull/3559/files#r44248843 for removing `bootstrap`, `json2` and `jquery` files?